template/src/navigation/CustomTabBar.tsx

```diff
@@ -1,13 +1,12 @@
-/**
- * Created by Hong HP on 11/17/19.
- */
-
-import {StyleSheet, Text, TouchableOpacity, View} from 'react-native'
+import {BottomTabBarProps} from '@react-navigation/bottom-tabs'
 import React from 'react'
+import {StyleSheet, Text, TouchableOpacity, View} from 'react-native'
 import {SafeAreaView} from 'react-native-safe-area-context'
+import {IItemTabBar} from './types'
 
-function CustomTabBar({routeName, navigation}) {
-  function renderItem(item, index) {
+const CustomTabBar: React.FC<BottomTabBarProps> = props => {
+  const {navigation} = props
+  function renderItem(item: IItemTabBar, index: number) {
```

Comment:
remove index: number, index is defined but never used

Reply:
Updated